presto-orc/src/main/java/com/facebook/presto/orc/writer/BooleanColumnWriter.java

```diff
@@ -68,16 +69,19 @@
 
     public BooleanColumnWriter(
             int column,
+            int sequence,
```

Comment:
In all the column writers, should we call sequence as dwrfSequence. As sequence is specific to DWRF format. Encryption specific to dwrf format is called dwrfEncryptor.